functions/knative/hello-world-go/README.md

```diff
@@ -6,6 +6,27 @@
 
 You can refer to the [Installation Guide](https://github.com/OpenFunction/OpenFunction#install-openfunction) to setup OpenFunction.
 
+## Run it locally
+
+Build the function locally
+
+```sh
+pack build func-helloworld-go --builder openfunction/builder-go:v2.3.0 --env FUNC_NAME="HelloWorld"  --env FUNC_CLEAR_SOURCE=true
```

Comment:
maybe the builder should have the go version as a suffix:
builder-go:v2.3.0-1.16

Reply:
done. by default we can create tag `v2.3.0` to the latest builder with the latest supported go version as well.